assets/i18n/qqq.json

```diff
@@ -220,6 +220,8 @@
 	"I18N_LEARNER_DASHBOARD_INCOMPLETE_SECTION": "Text for the incomplete section in the learner dashboard.",
 	"I18N_LEARNER_DASHBOARD_INTRO_MESSAGE_PART_ONE": "First part of the introductory message that appears on the learner dashboard when there is no activity.",
 	"I18N_LEARNER_DASHBOARD_INTRO_MESSAGE_PART_TWO": "Second part of the introductory message that appears on the learner dashboard when there is no activity.",
+	"I18N_LEARNER_DASHBOARD_NO_ACTIVITY_IN_COLLECTION": "Text displayed in the learner dashboard. - This text is shown when the user hasn't played a collection before and checking incomplete section",
```

Comment:
... and is checking the 'incomplete collections' section.

Ditto below (i.e., do a similar change). Otherwise it won't be clear to translators what is required.

Reply:
Done!